### PR TITLE
graduate PodLifecycleSleepAction to beta (KEP 3960)

### PR DESCRIPTION
**Pull request by AxeZhan**
This should only involves the stage change of the feature gate.

/hold 
Wait for https://github.com/kubernetes/kubernetes/pull/122456

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Hi. Please also revise https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#hook-handler-implementations (the feature gate is now enabled by default).

**Comment by AxeZhan** [comment]
> Hi. Please also revise https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#hook-handler-implementations 

Hmmm, copied from  `#hook-handler-implementations `
```
The "Sleep" action is available when the [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) PodLifecycleSleepAction is enabled.
```
Do we need to mention the feature gate is enabled default now? As user can see the default value of a feature gate when they click the `feature gate` link.

**Comment by sftim** [comment]
> Do we need to mention the feature gate is enabled default now? As user can see the default value of a feature gate when they click the `feature gate` link.

Yes, please reword to imply that it's an opt out.

**Review by sftim — COMMENTED** [review]
LGTM for docs

File: content/en/docs/reference/command-line-tools-reference/feature-gates/pod-lifecycle-sleep-action.md [hunk 1]
```diff
@@ -9,5 +9,8 @@ stages:
   - stage: alpha 
     defaultValue: false
     fromVersion: "1.29"
+  - stage: beta 
+    defaultValue: true
+    fromVersion: "1.30"
```
Comment (dipesh-rawat) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/pod-lifecycle-sleep-action.md:14]: Adding `toVersion` to the previous stage would enhance consistency with other feature gate files. 
```suggestion
    fromVersion: "1.29"
    toVersion: "1.29"
  - stage: beta 
    defaultValue: true
    fromVersion: "1.30"
```

Comment (AxeZhan) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/pod-lifecycle-sleep-action.md]: Yes, I missed this🤦. Thanks.

**Comment by AxeZhan** [comment]
/hold cancel
As https://github.com/kubernetes/kubernetes/pull/122456 has been merged.

**Comment by Princesso** [comment]
Hello @AxeZhan  :wave: please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before Tuesday March 12th 2024 18:00 PST. Thank you!

**Comment by katcosgrove** [comment]
Hey @AxeZhan! Just a reminder that this needs to be reviewed and ready to merge by [Docs Freeze](https://github.com/kubernetes/sig-release/blob/master/releases/release_phases.md#docs-freeze) on March 26, including a technical review. An [exception request](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md) will be required if you cannot meet that deadline. Thank you!

cc: @bobbypage @mrunalp @thockin @SergeyKanzhelev

**Comment by AxeZhan** [comment]
/cc @thockin @kannon92 
Can you guys give a review from sig-node side first?

**Review by kannon92 — COMMENTED** [review]
/lgtm

**Comment by Princesso** [comment]
Hello @AxeZhan  :wave:! I'm reaching out from the Docs team. Just checking in as we approach Docs Freeze on Tuesday March 26th 18:00 PDT. This documentation appears to still be under review. To meet the Docs Freeze, this PR must have a technical review as well as lgtm and approve labels applied, without any unaddressed comments or concerns from SIG Docs. The status of this enhancement is marked as at risk for docs freeze. Thank you!

**Comment by Princesso** [comment]
Could you ple look at this one more time?
/assign @sftim @nate-double-u

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/approve

**Pull request merged**